api/api.yml

```diff
@@ -0,0 +1,177 @@
+          404:
+            description: Not Found if there is no concordance record for the {uuid}.
+          405:
+            description: Method Not Allowed if anything other than a GET, PUT or DELETE is received.
+          500:
+            description: Internal Server Error if there was an issue processing the records.
+          503:
+            description: Service Unavailable if it cannot connect to the cache storage.
+
+      put:
+        summary: Stores the concordance record for the provided concept {uuid}.
+        description: Expects body in json format. Expects {uuid} in the path and in the body to match. The UUID in the URL should be the primary object, if the distinction exists (eg. where the two objects are of the same type).
+          tags:
+            - Functional Endpoint
+        consumes:
+          - application/json; charset=utf-8
+        parameters:
+          - in: body
+              {
+                "uuid": "4f50b156-6c50-4693-b835-02f70d3f3bc0",
+                "concordedIds": ["7c4b3931-361f-4ea4-b694-75d1630d7746", "1e5c86f8-3f38-4b6b-97ce-f75489ac3113", "0e5033fe-d079-485c-a6a1-8158ad4f37ce"]
+              }
+        responses:
+          200:
+            description: Updated if the record was successfully stored.
+          201:
+            description: Created if the record was successfully stored.
+          400:
+            description: Bad Request if the payload is badly formatted or does not contain the required fields.
+          405:
+            description: Method Not Allowed if anything other than a GET, PUT or DELETE is received.
+          500:
+            description: Internal Server Error if there was an issue processing the records.
+          501:
+            description: Service Unavailable if it cannot connect to the cache storage.
+
+      delete:
+        summary: Deletes the concordance record for a {uuid}
+        description: Deletes the concordance record for a {uuid}
+          tags:
+            - Functional Endpoint
+        responses:
+          204:
+            description: No Content if the record was successfully deleted.
+          400:
+            description: Bad Request if the payload is badly formatted or does not contain the required fields.
+          404:
+            description: Bad request if {uuid} is not valid.
+          405:
+            description: Method Not Allowed if anything other than a GET, PUT or DELETE is received.
+          500:
+            description: Internal Server Error if there was an issue processing the delete.
+          501:
+            description: Service Unavailable if it cannot connect to the cache storage.
+
+  /concordances/__count:
+        get:
+          summary: Concordance records count.
+          description: Gets a count of the number of concordance records.
+          produces:
+            - text/plain
+        tags:
+          - Functional Endpoint
+          responses:
```

Comment:
Incorrect indentation here - additional tab. It should be at the same level as tags